TileStache/Goodies/VecTiles/transform.py

```diff
@@ -337,6 +337,16 @@ def place_ne_capital(shape, properties, fid, zoom):
     return shape, properties, fid
 
 
+def water_tunnel(shape, properties, fid, zoom):
+    tunnel = properties.get('tunnel')
+    if tunnel in ('yes', 'true'):
```

Comment:
There seems to be a wide diversity of values [in taginfo for tunnel](http://taginfo.openstreetmap.org/keys/tunnel#values), almost a third are tagged `tunnel=culvert` and a smaller number are `tunnel=building_passage`. While we probably don't (yet) care about all these different types of tunnel, it seems there's a lot more values indicating some sort of tunnel than indicating no tunnel. Might it be better to reverse the logic and say `if tunnel not in ('no', 'false', '0')`?